test/run-drun/bad-actor.mo

```diff
@@ -0,0 +1,12 @@
+
+// bad leading decs
+type T1 = {};
+type T2 = {};
+
+actor Self {
```

Comment:
Perhaps move to `test/fail`? Or could we later define a semantics to consider such outside declarations to be in the actor scope? Many people tend to think this to be natural style.